core/src/main/scala/org/http4s/otel4s/middleware/OtelMetrics.scala

```diff
@@ -164,29 +164,29 @@ object OtelMetrics {
     }
 
   private def createMetricsCollection[F[_]: Monad: Meter](
-      prefix: String,
+      kind: String,
       responseDurationSecondsHistogramBuckets: BucketBoundaries,
   ): F[MetricsCollection[F]] = {
     val requestDuration: F[Histogram[F, Double]] =
       Meter[F]
-        .histogram[Double](s"$prefix.request.duration")
+        .histogram[Double](s"http.$kind.request.duration")
         .withUnit("s")
-        .withDescription("Duration of HTTP server requests.")
+        .withDescription(s"Duration of HTTP $kind requests.")
         .withExplicitBucketBoundaries(responseDurationSecondsHistogramBuckets)
         .create
 
     val activeRequests: F[UpDownCounter[F, Long]] =
       Meter[F]
-        .upDownCounter[Long](s"$prefix.active_requests")
+        .upDownCounter[Long](s"http.$kind.active_requests")
         .withUnit("{request}")
-        .withDescription("Number of active HTTP server requests.")
+        .withDescription(s"Number of active HTTP requests.")
```

Comment:
The interpolation s" is unused:
```suggestion
        .withDescription(s"Number of active HTTP requests.")
        .withDescription("Number of active HTTP requests.")
```